hack/common.sh

```diff
@@ -815,15 +815,17 @@ function os::build::image() {
   local dockerfile="${3-}"
   local options="${OS_BUILD_IMAGE_ARGS-}"
   local mode="${OS_BUILD_IMAGE_TYPE:-imagebuilder}"
+
   if [[ "${mode}" == "imagebuilder" ]]; then
-    if [[ -n "$( which imagebuilder )" ]]; then
+    if os::util::find::system_binary 'imagebuilder'; then
       if [[ -n "${dockerfile}" ]]; then
         eval "imagebuilder -f '${dockerfile}' -t '${tag}' ${options} '${directory}'"
```

Comment:
@Miciah, is it possible to support arbitrary flags like this?

@smarterclayton do you expect those flags to contain spaces? In either case, this just smells like a massive anti-pattern here.